media/css/cms/flare26-tokens.css

```diff
@@ -71,6 +73,7 @@ Static values extracted from Figma that do not change based on context.
 
     /* Width and viewport variables */
     --token-width-desktop-content: 697px;
+    --token-width-max-content: 725px;
```

Comment:
I think this is semantically what `--token-width-desktop-content` was intended for.